tests/denops/testdata/dummy_dispatcher_plugin.ts

```diff
@@ -6,9 +6,9 @@ export const main: Entrypoint = (denops) => {
     test: async (...args) => {
       await delay(100);
       await denops.cmd(
-        `execute 'doautocmd <nomodeline> User' fnameescape('DummyDispatcherPlugin:TestCalled:${
-          JSON.stringify(args)
-        }')`,
+        `doautocmd <nomodeline> User DummyDispatcherPlugin:TestCalled:${
+          JSON.stringify(args).replaceAll(/[ \\"]/g, "\\$&")
```

Comment:
What is `\\$&`?

Reply:
- `\\` Literal backslash.
- `$&` Inserts the matched substring.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#specifying_a_string_as_the_replacement

It means escaping spaces and backslashes with a backslash.

Reply:
First I tried `fnameescape`, but it works on Windows but not on Linux or Mac, so I can't use it.